### DISCUSSION TITLE
Create new Project View filtering with AND, OR, NOT operators

### DISCUSSION THREAD

**Discussion by jwerre** [community/community#16044] [Projects and Issues]
Is there a way to filter Projects with the `AND`, `OR`, `NOT` operators? I tired to create a Project View with the filter `( repo:"x" OR repo:"y" ) AND -status:Done` but that didn't seem to work. I was looking at the [docs](https://docs.github.com/en/search-github/getting-started-with-searching-on-github/troubleshooting-search-queries) and they is suggest that these operators are possible but not much about how to use them:

> You can't construct a query using more than five AND, OR, or NOT operators

I found some more details on `NOT` [here](https://docs.github.com/en/search-github/getting-started-with-searching-on-github/understanding-the-search-syntax#exclude-certain-results) but I could only get the `-` to work not the `NOT` operator. Is it safe to say that the GitHub search syntax doesn't work for GitHub Projects?

**Comment by michael-hawker** [comment]
Related to https://github.com/github-community/community/discussions/15655

**Comment by dkfellows** [comment]
According to [the documentation](https://docs.github.com/en/issues/planning-and-tracking-with-projects/customizing-views-in-your-project/filtering-projects):

> Using multiple filters will act as a logical AND filter. For example, `label:bug status:"In progress"` will return items with the `bug` label and the "In progress" status. You can also provide multiple values for the same field to act as a logical OR filter. For example, `label:bug,support` will return items with either the `bug` or `support` labels. Projects does not currently support logical OR filters across multiple fields.

It's a shame that full OR remains unsupported; it means I can't filter to, say, the Issues and PRs where _either_ I'm assigned to work on them _or_ where I'm a reviewer (that is, all the items in the project where I'm actioned to do something with them). Or the equivalent for someone else. Especially as well as combining that with other filters (e.g., to exclude the closed issues).

**Comment by mcmire** [comment]
Agreed. This is a big limitation in my opinion. I end up having to create multiple views when I really just want one view. Seems like something that should be easy if filtering on multiple values of the same field is possible.

**Comment by TimGuendel** [comment]
We need this as well 👀

**Comment by bholbrook73** [comment]
I read #15655 and it seemed like commas acted like "or", but trying this to find tasks missing something:
`no:product , no:type , no:status , no:priority` didn't work at all. Would really love to be able to have additional logical operators.

**Comment by JohannKaspar** [comment]
same here

**Comment by ElliotKetchup** [comment]
I need this to filter all the issues with missing fields, something like:
`no:repo OR no:assignee OR no:label`
It's a shame this is not implemented

**Comment by Gergling** [comment]
I would also like the filter to include OR functionality.

**Comment by douglasg14b** [comment]
I get the feeling that projects is an abandoned feature at this point, it's missing so many fundamental features to make it competitive. Like filter/query operators for example....

**Reply by Gergling** [reply]
Fine. I'll build my own. With blackjack. And hooks.

**Comment by amenity** [comment]
+1.

**Comment by felix-kocht** [comment]
same problem

**Comment by inoa-adottori** [comment]
+1

**Comment by bodnet-dev** [comment]
+1

**Comment by DubG1** [comment]
+1 would be really nice since we have a kanban and id like to see issues where i am assigned OR PRs where i am the reviewer and this doesnt work currently

**Comment by sanobarsyed** [comment]
+1 how di i find issues that are not added to any epic?

**Comment by dushanwegner** [comment]
+1 for OR

**Comment by sfa-schleupen** [comment]
In case you missed it, they did it 🥳 (at least AND and OR, not NOT yet)
https://github.blog/changelog/2026-07-16-advanced-search-for-projects-is-generally-available/
